asset.cpp

```diff
@@ -42,13 +42,12 @@ Asset::~Asset()
  * @param triggerReason		Why the notification is being sent
  * @param message		The message to send
  */
-void Asset::notify(const string& notificationName, const string& triggerReason, const string& message)
+bool Asset::notify(const string& notificationName, const string& triggerReason, const string& message)
 {
-vector<Datapoint *>	datapoints;
-
+	vector<Datapoint *>	datapoints;
 	if (!m_ingest)
 	{
-		return;
+		return false;
```

Comment:
Can we log some message here why retuning false ?

Reply:
fixed